packages/node_modules/@webex/plugin-meetings/src/locus-info/parser.js

```diff
@@ -17,9 +17,12 @@ Parser.updateWithDeltaLocus = (deltaLocus, meeting) => {
       desync: true,
       locusUrl: meeting.locusUrl
     }).then((res) => {
-      res.body.baseSequence = deltaLocus.baseSequence; // TODO: verify baseSequence age and update as necessary
-      meeting.locusInfo.onFullLocus(res.body);
+      if (res && res.body && res.body.baseSequence) {
```

Comment:
Do we only want to call `.onFullLocus` if the response body has a `baseSequence` that we then override?

If we're always going to set the `baseSequence` to the one from `deltaLocus` we could do:
```suggestion
      if (res && res.body) {
```

Reply:
When we have a partial locus object . We get the full locus object and update the base sequence as next time when an event comes we can compare the baseSequence 

I am not sure if it was the `res.body` not found or the `res.body.baseSequence` I might have mis read . Will remove it now and add a TODO if we see that error again